src/SFA.DAS.EmployerIncentives.Infrastructure/Configuration/BusinessCentralApiClient.cs

```diff
@@ -9,6 +9,7 @@ public class BusinessCentralApiClient : IApimClientConfiguration
         public string SubscriptionKey { get; set; }
         public string ApiVersion { get; set; }
         public int PaymentRequestsLimit { get; set; }
+        public bool ObfuscateSensitiveData { get; set; }
```

Comment:
Should it be more specific?  i.e. ObfuscateULN ?

Reply:
I thought about that, but what if we want to hide other data like a name or something else. Though  more general name was better

Reply:
I think keep the name generic, in case we need to obfuscate other data in the future